07-reputation/02-managers.md

```diff
@@ -0,0 +1,18 @@
+## Managers(mother nodes)
```

Comment:
I suggest to remove "mother nodes" remark, or change "mother" to "parent". It may insult non-female manager nodes.